packages/manager/apps/pci-instances/src/routes/routes.tsx

```diff
@@ -96,6 +97,12 @@ const routes: RouteObject[] = [
           import('@/pages/instances/instance/Instance.page'),
         ),
       },
+      {
+        path: SECTIONS.edit,
+        ...lazyRouteConfig(() =>
+          import('@/pages/instances/instance/edit/Edit.page'),
+        ),
+      },
```

Comment:
If we plan to use the migration I don't think this is useful, the link is sufficient I think

Reply:
I'll keep it to prevent from QA false negative, and to initiate the work on next pages